Source/Fuse.Controls.Panels/Layouts/WrapLayout.uno

```diff
@@ -135,6 +153,8 @@ namespace Fuse.Layouts
 			var placements = new float4[elements.Count];
 			//minorMaxSize in each major row, assinged per element
 			var minorSizes = new float[elements.Count];
+			// save the available space for each row
+			var majorRest = new Dictionary<float, float>();
```

Comment:
I don't understand how this could be a problem with precision? I'm taking the values as is. No arithmetic on it, so it should literally be the same value. 

So the idea is that `majorRest` will store the remaining value of each row. This could be done like `placements`, but it would be more involved, because several elements share the same row. So if we would like to do this we would save the index of the last time we changed row, and loop over the elements on this row, and update the available space for each of these. It's possible, but I like the current solution better.